desktop/cmp/filechooser/FileChooser.ts

```diff
@@ -81,6 +82,7 @@ export const [FileChooser, fileChooser] = hoistCmp.withFactory<FileChooserProps>
 
         return hbox({
             ref,
+            'data-testid': testId,
```

Comment:
Not needed (and confusing to switch to data-testid here) - just let props flow through